File: api/src/Fountain.ts


export interface Fountain {
    readonly id: string;
    readonly tObject: string;
    readonly modele: string;
    readonly numVoie: string;
    readonly voie: string;
    readonly commune: string;
    readonly disponibility: boolean;
    fav: boolean;
    readonly geoPoint2d: number[];
  }

File: android/app/src/main/java/com/ismin/csproject/Fountain.kt
package com.ismin.csproject

import java.io.Serializable

data class Fountain(
    val id: String,
    val tObject: String,
    val modele: String,
    val numVoie: String,
    val voie: String,
    val commune: String,
    val disponibility: Boolean,
    val fav: Boolean,
    val geoPoint2d: List<Double>
) : Serializable